README.md

```diff
@@ -2,16 +2,216 @@
 
 #### Table of Contents
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
-3. [Setup - The basics of getting started with iis](#setup)
-    * [Setup requirements](#setup-requirements)
-    * [Beginning with iis](#beginning-with-iis)
-4. [Usage - Configuration options and additional functionality](#usage)
-5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+1. [Description](#description)
+2. [Setup - The basics of getting started with IIS](#setup)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+    * [Facts](#facts)
+    * [Types/Providers](#types/providers)
+        * [iis_application_pool](#iis_application_pool)
+        * [iis_site](#iis_site)
 5. [Limitations - OS compatibility, etc.](#limitations)
 6. [Development - Guide for contributing to the module](#development)
 
-## Overview
+## Description
 
-This module adds a new provider capable of executing iis commands.
+This module provides a provider to manage IIS application pools and IIS sites.
```

Comment:
Might be better to say `adds` instead of `provides`, as it sounds redundant. I would also put sites before pools, if we are going to list things, as thats order of importance.